plugins/ilert/src/components/StatusPagePage/StatusPagesTable.tsx

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+import { TableState } from '../../api';
+import { StatusPage } from '../../types';
+import { VisibilityChip } from './VisibilityChip';
+
+import { Table, TableColumn } from '@backstage/core-components';
+import { StatusPageActionsMenu } from '../StatusPage/StatusPageActionsMenu';
+import { StatusPageLink } from '../StatusPage/StatusPageLink';
+import { StatusPageURL } from '../StatusPage/StatusPageURL';
+import { StatusChip } from './StatusChip';
+
+const useStyles = makeStyles(theme => ({
+  empty: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+export const StatusPagesTable = ({
+  statusPages,
+  tableState,
+  isLoading,
+  onChangePage,
+  onChangeRowsPerPage,
+  compact,
+}: {
+  statusPages: StatusPage[];
+  tableState: TableState;
+  isLoading: boolean;
+  setIsLoading: (isLoading: boolean) => void;
+  onChangePage: (page: number) => void;
+  onChangeRowsPerPage: (pageSize: number) => void;
+  compact?: boolean;
+}) => {
+  const classes = useStyles();
+
+  // const xsColumnStyle = {
+  //   width: '5%',
+  //   maxWidth: '5%',
+  // };
```

Comment:
clean up comments? Plus the other comments regarding type argument to TableColumn and props to makeStyles